packages/dialtone-icons/src/keywords.json

```diff
@@ -2298,4 +2308,4 @@
       ]
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Do you have a json linter enabled? I think this should have the blank line at the bottom, I had issues with this previously on releases.

Maybe your linter is removing that line or we have something missconfigured.

Reply:
Oh, thanks! Not sure why it got deleted, I don't have any json linter 😖 
Fixed in the lastest commit.